### PR TITLE
pd-ctl: retry when keyspace group manager not initialized

### LINKED ISSUE #7441: pd-ctl: show keyspace returns "keyspace manager is not initialized"
### What did you do?

start a cluster with tiup playground, then attempt to retrieve the default keyspace's meta with
`./pd-ctl keyspace show name DEFAULT`

### What did you expect to see?
`DEFAULT` keyspace's information is returned.

### What did you see instead?
```
Failed to get the keyspace information:  [500] "keyspace manager is not initialized"
```

### What version of PD are you using (`pd-server -V`)?
`latest` from master branch (commit: 4e9240a626855a66c849b753f34a4af3656e767b)

### PR DESCRIPTION
**Pull request by AmoebaProtozoa**
### What problem does this PR solve?

Issue Number: Close #7441

### What is changed and how does it work?

Make `refresh_group_id` optional when using show command on keyspace name.
This avoids `keyspace group uninitialized` error when pd is not running under api mode.

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: tools/pd-ctl/pdctl/command/keyspace_command.go [hunk 1]
```diff
@@ -87,7 +89,16 @@ func showKeyspaceNameCommandFunc(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	resp, err := doRequest(cmd, fmt.Sprintf("%s/%s?force_refresh_group_id=true", keyspacePrefix, args[0]), http.MethodGet, http.Header{})
```
Comment (lhy1024) [inline tools/pd-ctl/pdctl/command/keyspace_command.go:90]: maybe we can keep `force_refresh_group_id=true` as default, but retry again when meet `groupManagerUninitializedErr`

Comment (lhy1024) [reply tools/pd-ctl/pdctl/command/keyspace_command.go:90]: If no `force_refresh_group_id=true`, there will be inconsistent

Comment (AmoebaProtozoa) [reply tools/pd-ctl/pdctl/command/keyspace_command.go:90]: Ok, that sounds reasonable. I have change the force_refresh_group_id to default and added retry when it's groupManagerUnintalizedErr

File: tools/pd-ctl/pdctl/command/keyspace_command.go [hunk 2]
```diff
@@ -87,7 +89,21 @@ func showKeyspaceNameCommandFunc(cmd *cobra.Command, args []string) {
...
+	if err != nil {
+		cmd.PrintErrln("Failed to parse flag: ", err)
+		return
+	}
+	url := fmt.Sprintf("%s/%s", keyspacePrefix, args[0])
+	if refreshGroupID {
+		url += "?force_refresh_group_id=true"
+	}
+	resp, err := doRequest(cmd, url, http.MethodGet, http.Header{})
+	// Retry without the force_refresh_group_id if the keyspace group manager is not initialized.
```
Comment (lhy1024) [inline tools/pd-ctl/pdctl/command/keyspace_command.go:102]: Do we need to add a test for this case?

Comment (AmoebaProtozoa) [reply tools/pd-ctl/pdctl/command/keyspace_command.go:102]: added a specific case for this, PTAL

**Review by lhy1024 — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

**Pull request merged**